muzak.js

```diff
@@ -24,15 +24,8 @@ exports.handler = function (event, context) {
 
     try {
 
-        console.log("event.session.application.applicationId=" + event.session.application.applicationId);
         console.log("Event is %j", event);
 
-        // Limit access to only the configured application ID
-
-        if (event.session.application.applicationId !== config.alexaAppID) {
```

Comment:
tl;dr: I removed this to make it possible to run tests using claudia. 

I'm guessing this was added for security purposes, which makes sense. However, I'd argue that the application security is better handled by the AWS IAM role and the triggers set up to execute the Lambda function, which is only the Alexa Skills kit.  I don't see that this check in muzak.js adds any additional security, but it does make it much more different to run tests. 

With this check in place, the applicationId must be present in the request body, which means it needs to be part of any testing fixtures we use, which means that the testing fixtures can't simply be static json files - they'd have to be programmatically generated using the `config.js` file. It's do-able, but I didn't think it was worth that effort since I'm not sure this really adds any security. 

What do you guys think?